ant/org.eclipse.ant.tests.ui/test plugin/org/eclipse/ant/tests/ui/testplugin/TestAgainExceptionRule.java

```diff
@@ -0,0 +1,41 @@
+package org.eclipse.ant.tests.ui.testplugin;
```

Comment:
Missing license header

Reply:
Added in 6ed46f883aec1f280fd75ec75ff72dd8a343cbf5.